
Raw Benchmark Output
====================

Running Convolution 3D on Octuplets using NumPy/CPU
    commit: `#c0b56884ebe67fcb871d2c798370a2df66688b76 <https://bitbucket.org/bohrium/bohrium/commits/c0b56884ebe67fcb871d2c798370a2df66688b76>`_,
    time: 2015-03-20 04:06:13.551675.

    command: ``python benchmark/Python/convolve_3d.py --size=100 --bohrium=False``

Run 00
~~~~~~
    stdout::

        N/A

    stderr::

        python: can't open file 'benchmark/Python/convolve_3d.py': [Errno 2] No such file or directory
        



Run 01
~~~~~~
    stdout::

        N/A

    stderr::

        python: can't open file 'benchmark/Python/convolve_3d.py': [Errno 2] No such file or directory
        



Run 02
~~~~~~
    stdout::

        N/A

    stderr::

        python: can't open file 'benchmark/Python/convolve_3d.py': [Errno 2] No such file or directory
        



